media-tv/mythtv/mythtv-32.0.ebuild

```diff
@@ -0,0 +1,427 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8,9} )
+
+inherit eutils flag-o-matic python-any-r1 qmake-utils readme.gentoo-r1 systemd user-info
+
```

Comment:
Eutils should be droppable, mah need to replace with split eclasses depending on functionality used